com.bitwarden.desktop.yaml

```diff
@@ -12,7 +12,7 @@ sdk-extensions:
 finish-args:
   - --share=ipc
   - --share=network
-  - --socket=x11
+  - --socket=fallback-x11
```

Comment:
that mean on the wayland session there is no display.

Reply:
Ah, good catch, thank you!

Reply:
Ah, misunderstood this at first. Moved to only having `x11` now with the blocking PR added as a comment. As soon as that is fixed, will transition to wayland + fallback-x11.